src/main/java/com/suse/saltstack/netapi/calls/modules/Pkg.java

```diff
@@ -188,4 +188,26 @@ private Pkg() { }
         return new LocalCall<>("pkg.info_available", Optional.of(Arrays.asList(packages)),
                 Optional.empty(), new TypeToken<Map<String, Info>>(){});
     }
+
+    public static LocalCall<Boolean> upgradeAvailable(String packageName) {
+        LinkedHashMap<String, Object> args = new LinkedHashMap<>();
+        args.put("package", packageName);
+        return new LocalCall<>("pkg.upgrade_available", Optional.empty(), Optional.of(args),
+                new TypeToken<Boolean>(){});
+    }
+
+    public static LocalCall<String> latestVersion(String packageName) {
+        LinkedHashMap<String, Object> args = new LinkedHashMap<>();
+        args.put("package", packageName);
+        return new LocalCall<>("pkg.latest_version", Optional.empty(), Optional.of(args),
+                new TypeToken<String>(){});
+    }
+
+    public static LocalCall<Map<String, String>> latestVersions(String firstPackageName,
+            String secondPackageName, String...packages) {
+        return new LocalCall<>("pkg.latest_version",
+                    Optional.of(Arrays.asList(firstPackageName,
+                            secondPackageName, packages)),
+                Optional.empty(), new TypeToken<Map<String, String>>(){});
+    }
```

Comment:
Unified the two methods name with `latestVersion()` and fixed indentation.
We could also unify them in a unique method, using varargs. But then, inside the method, we should differentiate the return type based on the cardinality of the argument (`packageName`).
What do you think?